debian/postinst

```diff
@@ -72,6 +75,34 @@ configure)
 		echo "ERROR: postgres id $(id -u postgres) != 65437" >&2
 		exit 1
 	fi
+
+	# We are switching to MAC address based network interface names only in AWS environment
+	# from 6.0.10.0 onwards. To handle this change, we leverage the default 80-net-setup-link.rules udev
+	# rules file. We make a copy of this file under /etc and then change the naming policy to MAC address
+	# based names. Please see DLPX-75209 for more context on this change.
+	# Though unlikely, it is still remotely possible that the /usr/lib/udev/rules.d/80-net-setup-link.rules
+	# file either gets removed from the base OS distribution or gets replaced with another file to serve
+	# the exact/similar purpose. We must detect and inspect such a change and update the renaming logic if
+	# needed. Ideally, we should also detect changes in the contents of this file but we doubt there is an
+	# obvious way of accomplishing this today.
+	if [[ ! -e "$DEFAULT_UDEV_NET_SETUP_LINK_DIR"/80-net-setup-link.rules ]]; then
```

Comment:
are we sure this file will exist on non-aws platforms? what package provides it?

Reply:
I checked on ESX and this file is present. I could check on OCI and Azure but will need to figure out how to create an engine on these platforms.

The dkpg query for this file does not return a package name. Is there another way to check out the package name ?
```
delphix@DLPX-75209-TEST-20220912-1:~$ dpkg -S /usr/lib/udev/rules.d/80-net-setup-link.rules 
dpkg-query: no path found matching pattern /usr/lib/udev/rules.d/80-net-setup-link.rules
```

Reply:
Hm, yea.. I'd have to do some digging to see how/when it's generated.. seems to be some dynamically generated file.. I guess I'm OK landing without a double check on the other platforms.. presumably, if it doesn't exist, we'll catch that via a failed upgrade test on those platforms.

Reply:
@naveen-singh-delphix , the dpkg command isn't finding that path because the packaging defines it as `/lib/udev/...` (`/lib` is a symbolic link to `/usr/lib`):
```
root@guild:/export/home/delphix# dpkg -S /lib/udev/rules.d/80-net-setup-link.rules
udev: /lib/udev/rules.d/80-net-setup-link.rules
```

Reply:
Thanks @sebroy.